templates/layout-product-variation-slider.php

```diff
@@ -16,7 +11,12 @@
     ?>
       <li>
         <figure class="gallerya__image">
-          <?= wp_get_attachment_image($image->ID, apply_filters('gallerya/image_size_product_variation_slider', $slider_image_size), FALSE, $index ? [] : $image_attr) ?>
+          <?= wp_get_attachment_image($image->ID, apply_filters('gallerya/image_size_product_variation_slider', $slider_image_size), FALSE, $index ? [
+            'src' => 'data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7',
+            'srcset' => 'data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7',
+            'data-flickity-lazyload-src' => wp_get_attachment_image_url($image->ID, apply_filters('gallerya/image_size_product_variation_slider', $slider_image_size)),
```

Comment:
Shouldn't there be a single URL, but calling the same filter and passing the same value will probably result in two identical results. 🤔

Reply:
Good point, I'll use a variable.